100-Resources/200-Contributing/200-Contribute-to-Docs.mdx

```diff
@@ -43,13 +44,35 @@ To ensure your pull request passes checks, apply the linter by navigating to the
 
 ### Troubleshoot linting
 
-To fix issues from linting, locate the exact error message that references the specific number. Examine the code or documentation related to the error and ensure the syntax and usage align with coding and documentation standards.
+To fix issues from linting, locate the exact error message that references the specific line number of tf the error. Examine the code or documentation related to the error and ensure the syntax and usage align with coding and documentation standards.
 If the error persists, contact us in [Discord](https://xata.io/discord) or use community resources for guidance.
 
+After adding and committing code changes, the linter may identify errors. For instance, in the following example, an error is found on **line 62**, along with four additional warnings.
+
+![Linting errors](images/linting-error.png)
+
+To resolve the issue, go to **line 62** in your code, correct the error, then add, commit, and push the changes to create a pull request.
+
 ## Build preview
 
 After your pull request clears all checks, Vercel creates a build preview. The Xatabot then adds a link to this preview in your pull request. To review the changes you've made, click on the link, navigate to the **Docs** tab, and search for the page you've updated.
 
+![Preview changes](images/preview-links.png)
+
+### Preview changes in Visual Studio Code
+
+You can use Visual Studio Code (VS Code) to write documentation. VS Code includes a built-in Markdown previewer that you can use to view your changes locally.
+
+To access the Markdown Preview in Visual Studio Code, follow these steps:
+
+Open the Markdown file you want to preview. Select from the following methods based on your operating system:
+
+- **Mac OS:** Press **⌘** + **Shift** + **P**.
+- **Linux:** Press **Ctrl** + **Shift** + **P**
+- **Windows:** Press **Ctrl** + **Shift** + **P**
+
+Type "Markdown Preview" and select **Markdown: Open Preview** from the list that appears. Click the **Open Preview** icon located in the top right corner of the editor title bar or right-click the Markdown file and select **Open Preview** from the context menu.
```

Comment:
We could make clear that Vs code is not the only tool available, they can use any editor that has markdown preview.